### PR TITLE
PixelPaint: Make Wand Select tool select correctly on moved layers

### PR DESCRIPTION
**Pull request by snooze6214**
Before this commit, when the wand select tool was used on a layer that was moved, it would make the selection relative to the image, and not relative to the layer. 

Before this commit:

https://user-images.githubusercontent.com/7570033/196451756-17b6aed1-c64c-4372-a818-a4afdacaf638.mp4

After this commit:

https://user-images.githubusercontent.com/7570033/196451810-e0099f1d-2272-451c-ae1e-4d4164a0fe1f.mp4

### CODE REVIEW

File: Userland/Applications/PixelPaint/Tools/WandSelectTool.cpp [hunk 1]
```diff
@@ -46,8 +46,10 @@ void WandSelectTool::on_mousedown(Layer* layer, MouseEvent& event)
     if (!layer->rect().contains(layer_event.position()))
         return;
 
+    Gfx::IntPoint selection_offset = layer->relative_rect().top_left();
```
Comment (linusg) [inline Userland/Applications/PixelPaint/Tools/WandSelectTool.cpp:49]: `auto`

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Applications/PixelPaint/Tools/WandSelectTool.cpp (hunk 1 above)
Comment (snooze6214) [reply Userland/Applications/PixelPaint/Tools/WandSelectTool.cpp]: Done!

**Review by linusg — APPROVED** [review]

**Comment by tslater2006** [comment]
I actually have this fixed as a small fix in PR #15113, it just hasn't landed yet. https://github.com/SerenityOS/serenity/pull/15113/commits/5d37afbafffe8b0550388111e624f0b7d86dbef7

**Comment by tslater2006** [comment]
Might be easier to just land this for the fix and I can rebase my pending PR, would make it a bit cleaner anyways. Thanks for finding this and fixing @snooze6214 :)

**Pull request merged**